etc/nginx.conf

```diff
@@ -0,0 +1,38 @@
+worker_processes auto;
+daemon off;
+pid /tmp/nginx.pid;
+error_log  /dev/stderr;
+
+worker_rlimit_nofile 4096;
+
+events {
+  worker_connections 2048;
+}
+
+http {
+  # include /etc/nginx/mime.types;
```

Comment:
Did you notice that master is different: https://github.com/aws/sagemaker-containers/blob/master/src/container_support/etc/nginx.conf#L13